commit 165e7ad459fbba2f89708fba04a55bb3981e884c
Author: Szabolcs Nagy <szabolcs.nagy@arm.com>
Date:   Wed Feb 2 14:03:58 2022 +0000

    Fix elf/tst-audit25a with default bind now toolchains
    
    This test relies on lazy binding for the executable so request that
    explicitly in case the toolchain defaults to bind now.
    
    (cherry picked from commit 80a08d0faa9b224019f895800c4d97de4e23e1aa)

diff --git a/elf/Makefile b/elf/Makefile
index 0ab3e885f5e35671..9e4e056938a75ddb 100644
--- a/elf/Makefile
+++ b/elf/Makefile
@@ -2133,6 +2133,7 @@ $(objpfx)tst-audit25a: $(objpfx)tst-audit25mod1.so \
 		       $(objpfx)tst-audit25mod2.so \
 		       $(objpfx)tst-audit25mod3.so \
 		       $(objpfx)tst-audit25mod4.so
+LDFLAGS-tst-audit25a = -Wl,-z,lazy
 $(objpfx)tst-audit25mod1.so: $(objpfx)tst-audit25mod3.so
 LDFLAGS-tst-audit25mod1.so = -Wl,-z,now
 $(objpfx)tst-audit25mod2.so: $(objpfx)tst-audit25mod4.so
